src/librustc_typeck/diagnostics.rs

```diff
@@ -2348,8 +2348,8 @@ For information on the design of the orphan rules, see [RFC 1023].
 "##,
 
 E0211: r##"
-You used an intrinsic function which doesn't correspond to its
-definition. Erroneous code example:
+You used a function/type which doesn't correspond to its
```

Comment:
Which one doesn't have then ? And what "global" explanation would you give instead ?